features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.types.listing/public/js/listing.js

```diff
@@ -150,37 +125,47 @@ function loadDevices(searchType, searchParam){
     var deviceListingSrc = deviceListing.attr("src");
     var imageResource = deviceListing.data("image-resource");
     var currentUser = deviceListing.data("currentUser");
-    $.template("device-listing", deviceListingSrc, function (template) {
-        var serviceURL="/common/device_manager/device/type/all";
-        var successCallback = function (data) {
-            data = JSON.parse(data);
+
+    $('#ast-container').html("");
+
+    if(deviceTypesList.length > 0){
```

Comment:
This code is moved to components and fixed in [#e537e2d](https://github.com/wso2/carbon-device-mgt/commit/e537e2d841ca3c7946cd34e9da96f261e0ac94e1)